interactive_ai/workflows/geti_domain/common/jobs_common_extras/datumaro_conversion/import_utils.py

```diff
@@ -454,7 +454,7 @@ def get_label_to_ann_types(
                 if label_points:
                     # for keypoint annotations, we force the label_id to be 0 which is the first label in the dm dataset
                     label_idx_to_ann_types[0].add(dm_ann.type)
-                elif label_id:
+                elif label_id is not None:
```

Comment:
Suggestion: rename the variable to `label_idx` to make it clearer that it's an integer index instead of an objectid